bahmni_product/models/product.py

```diff
@@ -9,94 +9,18 @@
 class ProductProduct(models.Model):
     _inherit = 'product.product'
 
-    @api.depends('stock_quant_ids', 'stock_move_ids')
```

Comment:
Is this not being used anywhere, and hence deleted?

Reply:
yes,